.github/workflows/develop.yml

```diff
@@ -47,26 +39,13 @@ jobs:
             with:
                 python-version: ${{ matrix.python-version }}
 
-        -   name: Install system dependencies
+        -   name: Install Python dependencies
             run: |
-                sudo apt update
-                sudo apt install postgresql-12
-
-        -   name: Install python dependencies
-            continue-on-error: true
-            id: install
-            run: |
-                pip install --upgrade setuptools
+                pip install --upgrade pip setuptools wheel
                 pip install git+https://github.com/aiidateam/aiida-core@develop#egg=aiida-core[atomic_tools]
                 pip install -e .[tests]
                 reentry scan
 
-        -   name: Run pytest
-            continue-on-error: true
-            id: tests
-            run:
-                pytest -sv tests
-
         -   name: Slack notification
```

Comment:
Seems the workflow had been failing for a while now, but I didn't see any notifications in the Slack channel?

Reply:
Good point, not sure why because it used to work at some point, but looking at the syntax, it seems to rely on the `steps.install` and `steps.tests` identifiers. I had accidentally removed them so it would have failed, but they used to be there, so not sure why it wasn't working.